packages/ui/src/styles/global/typography.scss

```diff
@@ -8,14 +8,14 @@
 
 .link {
   color: var(--va-link-color);
-}
 
-.link-secondary {
-  color: var(--va-link-color-secondary) !important;
+  &--secondary {
+    color: var(--va-link-color-secondary) !important;
+  }
 }
 
 .link,
-.link-secondary {
+.link--secondary {
```

Comment:
This will cause refactoring somewhere. Let's let it be that ugly...

Reply:
returned as it was :)